src/app/qgsidentifyresultsdialog.cpp

```diff
@@ -820,9 +828,17 @@ QgsIdentifyResultsFeatureItem *QgsIdentifyResultsDialog::createFeatureItem( QgsV
       }
       else
       {
+        attrItem->setData( 1, REPRESENTED_VALUE_ROLE, representedValue );
+        
+        // Check if "Show full text" is enabled
+        const bool showFullText = QgsIdentifyResultsDialog::settingShowFullText->value();
+        
+        // Always create a label widget for consistent styling and text selection
+        QLabel *valueLabel = createStyledLabel( representedValue, showFullText );
+        attrItem->setText( 1, representedValue ); 
         attrItem->setData( 1, Qt::DisplayRole, representedValue );
```

Comment:
in the foundLinks == true branch we set the DisplayRole to QString() -- I believe that should also be done here, otherwise the transparent background for the label will result in overlapping text